index.html

```diff
@@ -0,0 +1,141 @@
+                </ul>
+            </section>
+        </nav>
+    </header>
+</nav>
+
+
+    <footer class="page-footer">
+        
+                <div class="container-fluid d-flex flex-row p-1 text-white">
+                   
+                    <div class="p-2">
+                        <a href="#"><img src="assets/logo/footerlogo.png" alt="" style="height: 50px;"></a>
+                    </div>   
+                    <div class="p-2 text-secondary">© 2021 NEWSWEEK DIGITAL LLC</div>
+
+
+                    <div class="p-2 text-secondary">
+                        <a href="#"><i class="fab fa-facebook-f"></i></a>
+                        <a href="#"><i class="fab fa-twitter"></i></a>
+                        <a href="#"><i class="fab fa-instagram"></i></a>
+                        <a href="#"><i class="fab fa-tumblr"></i></a>
+                        <a href="#"><i class="fab fa-linkedin-in"></i></a>
+                    </div>
+            
+                </div>
+
+                
+            <div class="row footer-two ">
+
+               <div class="row text-secondary">
+                    <ul class="d-flex flex-row list-inline ">
+                      <li class="p-1 m-1 text-secondary"><a href="">Editions: </a></li>
+                      <li class="p-1 m-1 small"><a href="">U.S. Edition</a></li>
+                      <li class="p-1 m-1 small"><a href="">日本</a></li>
+                      <li class="p-1 m-1 small"><a href="">Pakistan</a></li>
+                      <li class="p-1 m-1 small"><a href="">Polska</a></li>
+                      <li class="p-1 m-1 small"><a href="">România</a></li>
+                    </ul>
+                  </div>
+
+                <div class="row d-inline-block">
+                    <ul class="d-flex list-inline">
+                      <li class="p-1 m-1 small"><a href="">About Us</a></li>
+                      <li class="p-1 m-1 small"><a href="">Corrections</a></li>
+                      <li class="p-1 m-1 small"><a href="">Contact Us</a></li>
+                      <li class="p-1 m-1 small"><a href="">Editorial Guidelines</a></li>
+                      <li class="p-1 m-1 small"><a href="">Advertise</a></li>
+                      <li class="p-1 m-1 small"><a href="">Copyright</a></li>
+                      <li class="p-1 m-1 small"><a href="">Terms & Conditions</a></li>
+                      <li class="p-1 m-1 small"><a href="">Privacy Policy</a></li>
+                      <li class="p-1 m-1 small"><a href="">Cookie Policy</a></li>
+                      <li class="p-1 m-1 small"><a href="">Terms of Sale</a></li>
+                      <li class="p-1 m-1 small"><a href="">Archive</a></li>
+                      <li class="p-1 m-1 small"><a href="">Announcements</a></li>
+                    </ul>
```

Comment:
### Desktop
- [ ] The text in this part of the footer also should not have any text decoration
- [ ] The spacing in this section is also not quite right and the dividers between the text are missing. 
Your clone 👇🏾 
<img width="1426" alt="Screen Shot 2021-01-25 at 8 09 06 PM" src="https://user-images.githubusercontent.com/48804258/105753546-2f8db700-5f49-11eb-9463-8942abf68842.png">
The original page 👇🏾 
<img width="1321" alt="Screen Shot 2021-01-25 at 8 09 59 PM" src="https://user-images.githubusercontent.com/48804258/105753648-4f24df80-5f49-11eb-9b4e-63896693360d.png">

### Tablet & Mobile

- [ ] Your text in this section should wrap on the tablet and mobile screen sizes. Yours does not wrap and it is causing there to be a horizontal scroll.

Your clone 👇🏾 
**Tablet**
<img width="782" alt="Screen Shot 2021-01-25 at 8 20 24 PM" src="https://user-images.githubusercontent.com/48804258/105754778-c3ac4e00-5f4a-11eb-96ba-df17a206e387.png">
**Mobile**
<img width="431" alt="Screen Shot 2021-01-25 at 8 21 58 PM" src="https://user-images.githubusercontent.com/48804258/105754980-fbb39100-5f4a-11eb-9cd0-a98db3a3feeb.png">

The original page 👇🏾 
**Tablet**
<img width="776" alt="Screen Shot 2021-01-25 at 8 20 43 PM" src="https://user-images.githubusercontent.com/48804258/105754825-ce66e300-5f4a-11eb-8555-5bc9c6bb9bc6.png">
**Mobile**
<img width="428" alt="Screen Shot 2021-01-25 at 8 22 25 PM" src="https://user-images.githubusercontent.com/48804258/105755025-0b32da00-5f4b-11eb-8aa5-1f38407f3624.png">